app/scripts/components/common/map/style-generators/hooks.ts

```diff
@@ -159,10 +163,15 @@ export function useCMR({
 
   return assetUrl;
 }
-
-export function useWMS({ id, stacCol, stacApiEndpointToUse, onStatusChange }) {
+export function useWebMapService({
+  id,
+  stacCol,
+  stacApiEndpointToUse,
+  onStatusChange
+}) {
   const defaultBounds: [number, number, number, number] = [-180, -90, 180, 90];
-  const [wmsUrl, setWmsUrl] = useState('');
+  const [url, setUrl] = useState<string[]>();
```

Comment:
I am seeing the wrong use of this value as `string`, not `string[]` as it is intended. I wonder if it would be helpful to rename this variable as plural? (`urls`)